.travis.yml

```diff
@@ -28,9 +28,9 @@ before_install:
 
 install:
     - |
-      MINICONDA_URL="http://repo.continuum.io/miniconda"
+      MINICONDA_URL="https://repo.continuum.io/miniconda"
```

Comment:
@davidh-ssec this is the most important change from that re-rendering BTW. Continuum ended the `http` links and we had to move to `https`.